workshop-labs/docs/openshift101/00_overview.md

```diff
@@ -1,6 +1,30 @@
 # Overview
+Welcome!
 
-In these labs you will explore builds and deployments within the context of the Pathfinder OpenShift Platform.
+In these labs you will explore builds and deployments within the context of the Azure Redhat OpenShift Training Cluster.
 These labs will focus on a single 2-tier application based on the popular open source chat utility, Rocket.Chat.
 
-__Pleasse be aware the througout the labs, you will be guided to make common mistakes, and then guide you on how to fix them.__
\ No newline at end of file
+## Special Notes
+
+This training is being deliver in Openshift 4.3. The production Openshift cluster is currently in 3.11. 
+
+## Course Format
+
+This course has been designed to be run _at your own pace_. The lab materials have been designed so that you generally follow the happy bath for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques!
```

Comment:
```suggestion
This course has been designed to be run _at your own pace_. The lab materials have been designed so that you generally follow the happy path for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques!
```

Reply:
hahaha I like happy baths though. They are relaxing 👼